content/learning-paths/smartphones-and-mobile/profiling-unity-apps-on-android/6-unity-analyzer.md

```diff
@@ -0,0 +1,193 @@
+
+- Export can be used to save data in a format that the Analyzer can use directly
+
+- A graph of the first 300 frames of activity
+
+- Filters that let you search the data, e.g. by marker name or thread
+
+- Top 10 markers on the medium frame (i.e. a quick indication of which areas are taking the most time on the median frame)
+
+- Table of Marker Details for selected range
+
+- Frame summary (for frame range and timing ranges for those frames)
+
+- Thread summary details
+
+- Marker summary
+
+Markers relate to the profiling data markers as named in calls such as BeginSample.
+
+## Analyzing the unoptimized code
+
+The Analyzer will let you review performance data over multiple frames. Before you start you might want to find a frame that looks indicative of your application - and not an unusual outlier. 
+
+### Review the top 10 markers
+
+The Top 10 markers area can give a quick indication of problem areas if you are confident that the frames you have selected are representative of your application and not outliers.
+
+The (median) frame it chooses to highlight will be dependent on your frame selection. If you select a single frame, then the measurements will be for that single frame. If multiple frames are selected, it will choose the median frame for that selection.
+
+### Use the graph to select interesting frames
+
+The collision sample is reasonably consistent; the frame times don’t differ wildly between each other; though they do seem to get progressively worse. You can select a specific frame from the frame time graph by clicking anywhere on the graph.
+
+You can also reorder the data by frame duration:
+
+1. Right-click on the Analyzer graph to show a context menu
+
+1. Select Order by Frame Duration to toggle it on
+
+This may help you find a representative frame. Alternatively you could ask the Analyzer to find the median frame for you:
+
+1. Right-click on the graph again to show the context menu
+
+1. Select context menu option Select Median Frame
+
+The Analyzer will jump to the median frame. Notice the Marker Details table automatically updates with the relevant data.
+
+You can also reduce the data analysis to a specific range of frames.
+
+1. On the graph, left-click and drag to highlight the frames you want to analyze
+
+1. Hover your mouse over the selected frames in the graph to see a popup summary. The popup will tell you the frame duration of the exact frame you are hovering over, and also the total time of the selected frames.
+
+![Analyzer frame selection#center](images/analyzer-select-similar-frames.PNG "Figure 2. Selected range of similar frames from the unoptimized version.")
+
+You can see from the Marker Details table, the function CollisionCalculationScript.Update is high in the list.
+
+### Is the sample code GPU bound?
+
+We don’t think the collision sample is CPU bound but here’s how to tell using the Analyzer.
```

Comment:
GPU

Reply:
actually I don't think we explain the term "CPU/GPU bound" the first time we use it and it's probably not obvious to someone only just learning about profiling...